packages/standard/zarf.yaml

```diff
@@ -91,12 +97,6 @@ components:
     import:
       path: ../monitoring
 
-  # Authservice
```

Comment:
moved this up the list in the standard bundle as it should be deployed after keycloak.  and tests were failing as monitoring gets deployed before authservice if left alone.